pkg/proxy/iptables/proxier.go

```diff
@@ -913,33 +923,32 @@ func (proxier *Proxier) syncProxyRules() {
 					}
 					replacementPortsMap[lp] = socket
 				}
-			} // We're holding the port, so it's OK to install iptables rules.
```

Comment:
Reason to remove this comment?

Reply:
There's already a comment explaining the need to hold the port at the start of that block (https://github.com/danwinship/kubernetes/blob/07ead7d8/pkg/proxy/iptables/proxier.go#L894), so the comment at the end seemed redundant. I'll add it back if you want though.